+sig/playgroundPTB.m

```diff
@@ -19,17 +19,17 @@
   'TimerFcn', @process, 'Name', 'MainLoop');
 cp = hw.CursorPosition;
 
-vbox = uiextras.VBox('Parent', parent);
+vbox = uix.VBox('Parent', parent);
 
 % Get number of available screens
 nScreens = Screen('Screens');
-screenNum = iff(max(nScreens) > 2, 1, 0);
+screenNum = iff(nScreens(end) > 1, 1, 0);
 vc = Screen('OpenWindow', screenNum, 0, [0,0,1280,600], 32);
 Screen('FillRect', vc, 255/2);
 Screen('Flip', vc);
 
-btnbox = uiextras.HBox('Parent', vbox);
-vbox.Sizes = 30;
+btnbox = uix.HBox('Parent', vbox);
+%vbox.Sizes = 30;
```

Comment:
If this is no longer necessary perhaps just delete it

Reply:
done